### DISCUSSION TITLE
Excuse me, are the initial weights of features all 0?

### DISCUSSION THREAD

**Discussion by szuhzw** [CMU-SAFARI/Hermes#14] [Q&A] [answered]
Hello. I did not see the ocp_hermes.ini file initializing the weights. Are the initial weights of features all 0?

**Comment by rahulbera** [comment, marked as answer]
Hi @szuhzw yes, the weights are initialized to zero. See `inc/perc_pred.h:80`.

Please create a post in the discussion section of the repository (https://github.com/CMU-SAFARI/Hermes/discussions) for further questions.
